frontend/src/scenes/web-analytics/WebPropertyFilters.tsx

```diff
@@ -8,14 +8,18 @@ import { IconWithCount } from 'lib/lemon-ui/icons'
 import { LemonButton } from 'lib/lemon-ui/LemonButton'
 import { useState } from 'react'
 
-import { webAnalyticsLogic } from './webAnalyticsLogic'
+import { ProductTab, webAnalyticsLogic } from './webAnalyticsLogic'
 
 export const WebPropertyFilters = (): JSX.Element => {
-    const { rawWebAnalyticsFilters, preAggregatedEnabled } = useValues(webAnalyticsLogic)
+    const { rawWebAnalyticsFilters, preAggregatedEnabled, productTab } = useValues(webAnalyticsLogic)
     const { setWebAnalyticsFilters } = useActions(webAnalyticsLogic)
 
     const [displayFilters, setDisplayFilters] = useState(false)
 
+    if (productTab === ProductTab.MARKETING) {
```

Comment:
Nit: I'd prefer the parent handle this logic, rather than this component rendering null